### ISSUE TITLE
eBPF for Windows should work with the version of LLVM that ships with Visual Studio

### ISSUE THREAD

**Issue by Alan-Jowett** [microsoft/ebpf-for-windows#3678]
Microsoft Visual Studio ships with a version of LLVM (17.0.3). eBPF for Windows should work with this version and ideally our documentation / scripts should be updated to use this version instead of the older 11.0.1 version. 

The reason for adopting this version is that it has the support and backing of the Microsoft Visual Studio team and will be supported long term, unlike LLVM 11.0.1 which is unsupported.

**Comment by dthaler** [comment]
This looks like a duplicate of issue #1111

**Comment by saxena-anurag** [comment]
Blocked on verifier issue for branch folding.

**Comment by saxena-anurag** [comment]
@Alan-Jowett can you confirm if this is a duplicate of #1111

**Comment by saxena-anurag** [comment]
Cilium commit where verification fails today: https://github.com/cilium/cilium/commit/611dc2c0a4e0d9a5539f8ada5270148e176c4ebd

**Comment by Alan-Jowett** [comment]
Is it possible to share pre-compiled ELF files that are failing? We can then work with the verifier team to get to the root cause and get it fixed.

**Comment by Alan-Jowett** [comment]
Fixed
